app/controllers/email/VerifyEmailController.scala

```diff
@@ -48,99 +44,10 @@ class VerifyEmailController @Inject()(val emailVerificationService: EmailVerific
 
   implicit val ec: ExecutionContext = mcc.executionContext
 
-  def emailShow: Action[AnyContent] = (blockAgentPredicate andThen inFlightEmailPredicate) { implicit user =>
-
-    extractSessionEmail match {
-      case Some(email) => Ok(verifyEmailView(email, isContactPrefJourney = false))
-      case _ => Redirect(routes.CaptureEmailController.show())
-    }
-  }
-
   def emailSendVerification: Action[AnyContent] = blockAgentPredicate.async { implicit user =>
```

Comment:
Could you remove the `implicit user =>` from this action as SBT is raising it as a warning due to it now being unused